package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "",
   "main": "index.js",
-  "bin": "index.js",
+  "bin": "./bin/krabby.js",
```

Comment:
I was going from the package.json docs
https://docs.npmjs.com/files/package.json#bin

Happy to remove, am I looking in the wrong place?